statsmodels/discrete/tests/test_count_model.py

```diff
@@ -314,8 +344,8 @@ class TestZeroInflatedGeneralizedPoisson_predict(object):
     @classmethod
     def setup_class(cls):
         expected_params = [1, 0.5, 0.5]
-        np.random.seed(1234)
-        nobs = 200
+        np.random.seed(999)
+        nobs = 2000
```

Comment:
Since you are changing the seed and the nobs, why not just change nobs?

Reply:
I discovered the test failures by just changing the seed so wanted to leave a trace of it. 
That this PR is sparked by test failing from just a change is seed.